src/www/services_router_advertisements.php

```diff
@@ -348,9 +348,10 @@ function toggle_dns(toggle) {
                         $carplist[$ifname] = convert_friendly_interface_to_friendly_descr($ifname);
                       }
                     }
-                    foreach (get_configured_ip_aliases_list() as $vip => $ifname) {
-                      if ($ifname == $if && (is_linklocal($vip)))
-                        $aliaslist[$vip] = get_vip_descr($vip) . ' (' . $vip . ')';
+                    foreach (config_read_array('virtualip', 'vip') as $vip) {
+                      if ($vip['mode'] == 'ipalias' && $if == $vip['interface'] && (is_linklocal($vip['subnet']))) {
```

Comment:
let's drop the spurious use of parenthesis

```suggestion
                      if ($vip['mode'] == 'ipalias' && $if == $vip['interface'] && is_linklocal($vip['subnet'])) {
```